cmscontrib/updaters/update_29.py

```diff
@@ -35,6 +35,14 @@
 logger = logging.getLogger(__name__)
 
 
+def fix_text(t):
+    if t is None:
+        return []
+    t = json.loads(t)
```

Comment:
It is possible t was a single string and not a JSON-encoded array - so if loads fails you need to return [t].